### PR TITLE
Payment method details

### PR DESCRIPTION
**Pull request by shirmagen**

### CODE REVIEW

File: examples/client/Locomotion/src/Components/CardDetails/index.tsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+import i18n from '../../I18n';
+import PageHeader from '../PageHeader';
+import { PaymentMethodInterface } from '../../context/payments/interface';
+import { getTogglePopupsState } from '../../context/state';
+import PaymentsContext from '../../context/payments';
+import deleteIcon from '../../assets/delete.svg';
+import { DeleteIcon, DeleteText } from './styled';
+
+
+export default ({
```
Comment (avivbarniv) [inline examples/client/Locomotion/src/Components/CardDetails/index.tsx]: create variable and then export it
const CardDetails = ...

export default CardDetails


Also, add interface

File: examples/client/Locomotion/src/Components/CardDetails/index.tsx [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+              onPress={undefined}
+            >
+              {i18n.t('payments.cardDetails.outstandingBalanceText')}
+            </Card>
+            <LogoutContainer
+              onPress={async () => {
+                await onRemoveMethod(paymentMethod?.id);
+              }}
+            >
+              <View style={{
```
Comment (avivbarniv) [inline examples/client/Locomotion/src/Components/CardDetails/index.tsx]: change to styled component

File: examples/client/Locomotion/src/Components/CardRow/index.tsx [hunk 3]
```diff
@@ -146,7 +143,7 @@ export default (paymentMethod: any) => (
                     {capitalizeFirstLetter(paymentMethod.brand)}
                   </Type>
                 )}
-              {paymentMethod.lastFour ? <Description>{`**** ${capitalizeFirstLetter(paymentMethod.lastFour)}`}</Description> : null}
+              {paymentMethod.lastFour ? <Description>{getLastFourForamttedShort(paymentMethod.lastFour)}</Description> : null}
```
Comment (avivbarniv) [inline examples/client/Locomotion/src/Components/CardRow/index.tsx:153]: nice

File: examples/client/Locomotion/src/pages/Account/index.js [hunk 4]
```diff
@@ -68,7 +68,7 @@ const AccountHeader = () => {
   );
 };
 
-const Card = ({
+export const Card = ({
```
Comment (avivbarniv) [inline examples/client/Locomotion/src/pages/Account/index.js]: Extract it to different file with better naming

File: examples/client/Locomotion/src/pages/Payments/cardDetailUtils.ts [hunk 5]
```diff
@@ -0,0 +1,3 @@
+export const capitalizeFirstLetter = (string: string) => string?.charAt(0).toUpperCase() + string?.slice(1);
+export const getLastFourForamttedShort = (lastFour: string) => `**** ${capitalizeFirstLetter(lastFour)}`;
+export const getLastFourForamttedShortLong = (lastFour: string) => `**** **** **** ${capitalizeFirstLetter(lastFour)}`;
```
Comment (avivbarniv) [inline examples/client/Locomotion/src/pages/Payments/cardDetailUtils.ts]: remove Short

Comment (shirmagen) [reply examples/client/Locomotion/src/pages/Payments/cardDetailUtils.ts:3]: in choose payment method and credit card list we use short

File: examples/client/Locomotion/src/pages/Payments/index.js [hunk 6]
```diff
@@ -90,17 +70,6 @@ export default ({ navigation, menuSide }) => {
             />
           </CardContainer>
         )}
```
Comment (avivbarniv) [inline examples/client/Locomotion/src/pages/Payments/index.js:72]: Why?

**Pull request merged**